### PR TITLE
Added importing functionality in collaborative editing section. 

### PR DESCRIPTION
**Pull request by Vasanth-3519**
**Description:**
Added the importing functionality on collaborative editing mode in Typescript UG documentation.

### CODE REVIEW

File: ej2-javascript/spreadsheet/use-cases/collaborative-editing.md [hunk 1]
```diff
@@ -174,6 +174,64 @@ spreadsheet.appendTo('#spreadsheet');
...
+import { isNullOrUndefined } from "@syncfusion/ej2-base";
+
+// For signalR Hub connection
+const connection: signalR.HubConnection = new signalR.HubConnectionBuilder().withUrl('https://localhost:44385/hubs/spreadsheethub', { // localhost from AspNetCore service
+    skipNegotiation: true,
+    transport: signalR.HttpTransportType.WebSockets
+}).build();
+
+//Initialize the SpreadSheet control
+let spreadsheet: Spreadsheet = new Spreadsheet({
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/use-cases/collaborative-editing.md:195]: If you add directly to the MD file, Both JavaScript and TypeScript platform UG will display the Typescript code in all places. Need to add separate the code for JavaScript also.

https://ej2.syncfusion.com/javascript/documentation/spreadsheet/use-cases/collaborative-editing

https://ej2.syncfusion.com/documentation/spreadsheet/use-cases/collaborative-editing

File: ej2-javascript/code-snippet/spreadsheet/data-binding-cs2/index.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var query = new ej.data.Query().select([
 
     //Initialize DataManager.
     var data = new ej.data.DataManager({
-        url: 'https://js.syncfusion.com/demos/ejServices//wcf/Northwind.svc/Orders',
+        url:  'https://services.odata.org/V4/Northwind/Northwind.svc',
```
Comment (gobalakrishnans) [inline ej2-javascript/code-snippet/spreadsheet/data-binding-cs2/index.js:7]: it will work if you removed the Orders from the URL ??

**Review by gobalakrishnans — CHANGES_REQUESTED** [review]

File: ej2-javascript/code-snippet/spreadsheet/data-binding-cs2/index.js (hunk 2 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/data-binding-cs2/index.js:7]: Checked with the documentation and resolved.

File: ej2-javascript/spreadsheet/use-cases/collaborative-editing.md [hunk 3]
```diff
@@ -174,6 +174,64 @@ spreadsheet.appendTo('#spreadsheet');
...
+    openUrl: 'https://services.syncfusion.com/js/production/api/spreadsheet/open',
+    actionComplete: (args: any) => {
```
[the remark this answers is not in the document]
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/use-cases/collaborative-editing.md:197]: Checked and resolved

File: ej2-javascript/spreadsheet/use-cases/collaborative-editing.md [hunk 4]
```diff
@@ -174,6 +174,64 @@ spreadsheet.appendTo('#spreadsheet');
 
 ```
 
+## Perform import action for collaborative clients
+
+Using the `action` argument from the `actionComplete` event, you can identity whether the import action is performed or not. If the action is `import`, then you need to send the `response data` to the server and also update the same to the collaborative clients.
```
[the remark this answers is not in the document]
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/use-cases/collaborative-editing.md:179]: Checked and added the referencce

File: ej2-javascript/spreadsheet/use-cases/collaborative-editing.md (hunk 1 above)
Comment (babu-periyasamy) [reply ej2-javascript/spreadsheet/use-cases/collaborative-editing.md:195]: Checked and added sample code for javascript.

File: ej2-javascript/spreadsheet/js/use-cases/collaborative-editing.md [hunk 5]
```diff
@@ -164,4 +164,59 @@ connection.start().then(() => {
...
+        spreadsheet.openFromJson({ file: jsonData });
+    }
+    else {
+        // Update the action details to the connected clients.
+        spreadsheet.updateAction(model);
+    }
+});
+connection
+    .start()
+    .then(() => { // to start the server.
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/js/use-cases/collaborative-editing.md:218]: Standardize this comment.

**Pull request merged**